cmd/iaviewer/main.go

```diff
@@ -101,11 +103,15 @@ func PrintDBStats(db dbm.DB) {
 
 // ReadTree loads an iavl tree from the directory
 // If version is 0, load latest, otherwise, load named version
-func ReadTree(dir string, version int) (*iavl.MutableTree, error) {
+func ReadTree(dir string, version int, prefix []byte) (*iavl.MutableTree, error) {
```

Comment:
Please update the function comment.
The `iaviwer` will always set a prefix.

Reply:
ok